src/components/Button.jsx

```diff
@@ -1,32 +1,33 @@
-import React from 'react'
-import './button.css'
-import { Link } from 'react-router-dom'
+import React from "react";
+import "./button.css";
+import { Link } from "react-router-dom";
 
-const STYLES = ['btn--primary', 'btn--outline'];
-const SIZES = ['btn--medium', 'btn--large'];
+const STYLES = ["btn--primary", "btn--outline"];
+const SIZES = ["btn--medium", "btn--large"];
 
 export const Button = ({
-    children,
-    type,
-    onClick,
-    buttonStyle,
-    buttonSize
+  children,
+  type,
+  onClick,
+  buttonStyle,
+  buttonSize,
```

Comment:
This is destructured data that are passed to the <Button /> component as props. Destructured means, you're already assigning them as variables along the way instead of like, onClick = props.onClick. But you're not using any of the destructured data so remove them?